pages/user/resume.js

```diff
@@ -23,12 +23,13 @@ const ResumePage = () => {
           members:
         </p>
 
-        <div className="pt-3 flex flex-col flex-grow w-11/12 pb-1">
-          <div className="w-3/5 flex flex-col flex-grow">
-            <ResumeUpload resume={session?.user?.resume} />
-            <PDFViewer pdf={session?.user?.resume} />
-            <ResumeFeedback />
-          </div>
+        <div className="pt-3 flex flex-col w-11/12 pb-1">
+          <ResumeUpload resume={session?.user?.resume} />
```

Comment:
session?.user?.resume shouldnt be necessary. protected page handles it

Reply:
Same issue with 

https://github.com/acm-ucr/membership-portal/pull/489

![image](https://github.com/acm-ucr/membership-portal/assets/53993828/9ce471b2-b14d-4812-a011-99fa81a14bf3)